### PR TITLE
Bytes array, multiple annotations per frame SN-84,SN-85

### PR DESCRIPTION
**Pull request by ovalle15**
Updates to use bytes array for masks (video, image), and add examples of multiple notations per frame (video)

### CODE REVIEW

File: examples/annotation_import/image.ipynb [hunk 1]
```diff
@@ -430,23 +432,26 @@
     {
       "metadata": {},
       "source": [
-        "# Identifying what values in the numpy array correspond to the mask annotation\n",
-        "color = (255, 255, 255)\n",
-        "mask_data = lb_types.MaskData(url=\"https://storage.googleapis.com/labelbox-datasets/image_sample_data/raster_seg.png\")\n",
+        "### Raster Segmentation (Byte string array)\n",
+        "url = \"https://storage.googleapis.com/labelbox-datasets/image_sample_data/raster_seg.png\"\n",
+        "# url = \"https://storage.googleapis.com/labelbox-datasets/image_sample_data/color_mask.png\"\n",
```
Comment (apollonin) [inline examples/annotation_import/image.ipynb:437]: do we want to leave the comments here?

**Review by apollonin — APPROVED** [review]
This looks good to me, assuming we now "force" users to use im_bytes as a default input instead of url

**Comment by ovalle15** [comment]
@apollonin  Manu suggested we use image bytes as default.

File: examples/annotation_import/image.ipynb (hunk 1 above)
Comment (ovalle15) [reply examples/annotation_import/image.ipynb:437]: Does it make sense?  or not really and I should remove it?

Comment (apollonin) [reply examples/annotation_import/image.ipynb:437]: I mean looks like you have 
```
url = "https://storage.googleapis.com/labelbox-datasets/image_sample_data/color_mask.png"
# url = "https://storage.googleapis.com/labelbox-datasets/image_sample_data/color_mask.png"
```

^ these two lines in the notebook and I don't see why we need the seconds line. could be just a leftover?

Comment (ovalle15) [reply examples/annotation_import/image.ipynb:437]: AH - will remove that

Comment (ovalle15) [reply examples/annotation_import/image.ipynb]: Done

**Pull request merged**